### ISSUE TITLE
not possible to use progress bars in shiny modules - 'session' is not a ShinySession object

### ISSUE THREAD

**Issue by FrissAnalytics** [rstudio/shiny#1062]
Hi,

I think currently it is not possible to use progress bars inside Shiny modules as the session object inside a module is of class *session_proxy* and is not a *ShinySession* object. Minimal example:

```r

library(shiny)

testModuleUI <- function(id){
  ns <- NS(id)
  tagList(
    h1("module ui")
  )
}

testModule <- function(input, output, session){

  ns <- session$ns

  cat(class(session))
  
   observe({
     progress <- shiny::Progress$new(session = session)
     progress$set(message = "lorem ipsum...", value = 0.1)
     on.exit(progress$close())
   })

}

app <- shinyApp(
  ui = bootstrapPage(
    testModuleUI("id1")
  ),
  server = function(input, output, session) {
    callModule(module = testModule, id = "id1")
  }
)

runApp(app)

```

This minimal example generates the following error:

``` r
Error in public_bind_env$initialize: 'session' is not a ShinySession object.
```

**Comment by wch** [comment]
@jcheng5, this was fixed by 9c4d142, right?

**Comment by FrissAnalytics** [comment]
indeed, in shiny v0.13.0 progress bars now work in modules as well...
